modulus/models/gnn_layers/mesh_graph_mlp.py

```diff
@@ -26,11 +26,11 @@
 from .utils import CuGraphCSC, concat_efeat, sum_efeat
 
 try:
-    from apex.normalization import FusedLayerNorm
+    from transformer_engine import pytorch as te
 
-    apex_imported = True
+    te_imported = True
 except ImportError:
-    apex_imported = False
+    te_imported = False
```

Comment:
If we are going to add `transformer_engine` as an optional library then I think it should be included in the toml. Is `transformer_engine` included in the pytorch base container that we use?

Reply:
Yes, it is included in the pytorch base image > 23.10